incubator/hnc/internal/pkg/selectors/selectors.go

```diff
@@ -0,0 +1,70 @@
+// Package selectors contains exceptions related utilities, including check the validity of selector,
+// treeSelector, and noneSelector, and parsing these three selectors.
+package selectors
+
+import (
+	"errors"
+	"strings"
+
+	"github.com/go-logr/logr"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/labels"
+
+	api "sigs.k8s.io/multi-tenancy/incubator/hnc/api/v1alpha2"
+)
+
+func GetSelectorAnnotation(inst *unstructured.Unstructured) (string, bool) {
```

Comment:
Didn't we agree that an empty selector is the same as a missing selector? If so, we don't need to return  `(string, bool)`, we can just return a string, right?

Reply:
done